js-core-1/week-2/lesson.md

```diff
@@ -472,7 +479,27 @@ function secondMatchesAmy(array) {
 }
 ```
 
-### Exercise (20 mins)
+### Using if loops with arrays
+
+We can use the power of loops to run some code for each element in our array.
+
+When we do this say we **iterate** over an array.
+
+```js
+let daysOfWeek = ["Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"];
+
+for(let i = 0; i < daysOfWeek.length; i++){
```

Comment:
I think this is a good opportunity to highlight the difference between `const` and `let`. If everything uses `const` by default we can simply ask, what happens if you can `let i = 0` to `const i = 0`? They will get an error saying that you can't reassign `const` and there's your answer!